src/main/java/de/tum/in/www1/artemis/service/connectors/bamboo/BambooService.java

```diff
@@ -833,6 +833,13 @@ private void addFeedbackToResult(Result result, List<BambooBuildResultNotificati
         }
     }
 
+    /**
```

Comment:
This JavaDoc should be stored in the corresponding interface.

Reply:
This is now removed, since it is already documented in the superclass.